buildout.cfg

```diff
@@ -5,7 +5,7 @@ eggs = mutagentools
 versions = versions
 
 [versions]
-# blank for now
+ipython = 5.3.0
```

Comment:
Apparently [there is some way to fix this](http://stackoverflow.com/a/11505595/128967), as IPython 6 and above _will not_ install on Python < 3.3. It just flat out fails to install, so backwards compatibility is broken.

We could use the `mr.scripty` recipe to set this value to latest 5.x.x for Python < 3.3.